src/Message/Analytics/UserLogOnOffSessionMessage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Message\Analytics;
+
+use App\Message\Analytics\Helper\AnalyticsDataType;
+use DateTimeImmutable;
+use JsonSerializable;
+use Symfony\Component\Uid\Uuid;
+
+class UserLogOnOffSessionMessage extends AnalyticsMessageBase implements JsonSerializable
+{
+
+    public readonly int $userId;
+    public readonly string $userName;
+    public readonly int $gameSessionId;
+    public readonly int $countryId;
+
+    public function __construct(
+        bool $logOn,
+        DateTimeImmutable $timeStamp,
+        Uuid $serverManagerId,
+        int $userId,
+        string $userName,
+        int               $gameSessionId,
+        int               $countryId,
```

Comment:
consistent spacing fix
```suggestion
        int $gameSessionId,
        int $countryId,
```